### PR TITLE
Removes heat resistance from explorer suits and slightly tweaks the ash drake to compensate

### PR DESCRIPTION
**Pull request by Deafult-Profile**
## About The Pull Request

removes the heat protection from the explorer suit and reduces the heat of the ash drakes fire

## Why It's Good For The Game

miners aren't supposed to be able to tank as much fire as they want without taking damage using roundstart gear but they aren't supposed to take 60 burn damage from a tiny little bit of fire either >:(

## Changelog
:cl:
balance: removes the explorer suit's heat resistance
balance: lowers the ash drake's fire damage
:cl:

### CODE REVIEW

**Comment by Deafult-Profile** [comment]
**I almost orkposted**

**Comment by SweptWasTaken** [comment]
GoffOrk

**Comment by Tumesuo** [comment]
default profile, its a bot ban it

**Comment by Tumesuo** [comment]
orkcat

**Comment by yeeyeh** [comment]
why is your profile picture pants
image

**Comment by Deafult-Profile** [comment]
> why is your profile picture pants
> image

it is not pants, it's a man with long eyes and a sick handlebar mustache

**Comment by Naloacisamazing** [comment]
fuck orks

**Comment by Deafult-Profile** [comment]
fuck orks~

**Comment by Pumpkinoe** [comment]
wont this stop explorer suits from working as EVA gear on icebox station or am i head empty

**Comment by Tumesuo** [comment]
> wont this stop explorer suits from working as EVA gear on icebox station or am i head empty

>icebox

**Comment by Pumpkinoe** [comment]
Yeah I'm right this stops explorer suits from working as EVA gear for the snowy wastes on Icebox
Probably not intentional I'm guessing?

**Comment by ATH1909** [comment]
cringe

**Comment by ATH1909** [comment]
also, this isn't a fix, this is a balance change

**Comment by ATH1909** [comment]
imagine getting triggered over miners going on short spacewalks using their single-use healing items

**Comment by Pumpkinoe** [comment]
i don't think this would even fix anything since the explorer suit has the same cold resist as the winter coat (you could just use the mining winter coat instead)
and removing the heat resistance just makes it much worse at dealing with fire when getting dipped in lava/fighting an ash drake

**Comment by Deafult-Profile** [comment]
> i don't think this would even fix anything since the explorer suit has the same cold resist as the winter coat (you could just use the mining winter coat instead)
> and removing the heat resistance just makes it much worse at dealing with fire when getting dipped in lava/fighting an ash drake

I'll fix the cold resist but I'm pretty sure you're not supposed to be able to tank fire? hewwo? Just get good and bring a fire extinguisher

**Comment by Deafult-Profile** [comment]
swept please remove the thumbs up from my pr

**Comment by Pumpkinoe** [comment]
from every standpoints it makes sense that the miners getting sent to a fire-hell lava-ridden planet would get fire-resistant gear

**Comment by Deafult-Profile** [comment]
You might as well ask for them to be made lavaproof too, get good

**Comment by Pumpkinoe** [comment]
i mean, there are plenty other ways to tank fire but, hey, w/e

**Comment by Doosteroni** [comment]
Well you know what!! Why do the "engineers" get space proof gear? What about sec? No hardsuit for you. Let's just remove EVA at this point! Why does anybody need to go to space? While we are at it, let's get rid of medipens. Oooh those pesky miners will regret ever getting good gear hahaha. Lava? More like orange-colored chasm!

**Comment by Doosteroni** [comment]
But seriously why in the hell is this a fix? This is clearly a rebalance. I never do recall seeing "minerman can spacewalk ban he!"

**Comment by Deafult-Profile** [comment]
let's compromise and make it both

**Comment by Doosteroni** [comment]
Compromises? You sound like a moderate!

**Comment by Deafult-Profile** [comment]
also miners already have hardsuits that they can buy along with two no slowdown hardsuits from tendrils that they have a chance to get. :(

**Comment by Doosteroni** [comment]
Fair enough, I'm all for making the mining hardsuit less terrible garbage fire (It's the normal miner suit with more slowdown) so I will give you that one. Tendrils are really a gamble though so most miners will not be so fortunate.

**Comment by Deafult-Profile** [comment]
they might be a gamble but they're piss easy to do, it doesn't help that they give better gear than most megafauna do. Most of the time as miner you stay on lavaland/ice moon for most of the shift and kill shit so what do you so desperately need extreme cold protection for?

**Comment by Deafult-Profile** [comment]
wait I just forgot, I didn't actually change how cold works with it, I just argued for something that I didn't even change!

**Comment by Deafult-Profile** [comment]
oops

**Comment by ATH1909** [comment]
> they might be a gamble but they're piss easy to do, it doesn't help that they give better gear than most megafauna do. Most of the time as miner you stay on lavaland/ice moon for most of the shift and kill shit so what do you so desperately need extreme cold protection for?

don't you need cold protection for icebox (which got merged, btw)?

**Comment by Doosteroni** [comment]
Icebox is a pain in the ass and doesn't bode well with this because the madman who made the demonic portal spawnlists (Tendrils with extra fuck you and lag making) decided to also add adamantine slime cores as one of the spawnables along with a bunch of garbage. I doubt you are gonna get many actually good items from them along with the fact that A: they have no GPS signals if I am not mistaken and B: they spawn Demonic watchers which are the biggest pain in the ass ever. I do believe there is an open merge suggesting the removal of some of the hardsuits too so yeah, not a good combo.

**Comment by Doosteroni** [comment]
I'm pretty sure the PR I mentioned got closed so uhh just ignore that section at the bottom.

**Comment by Deafult-Profile** [comment]
I'd take an adamantine slime core over some plasmafire bomb that gets rid of all my other loot anyday, I've managed to accidentally set that off too much. I hope icebox gets some extra polish in the future but who knows

**Comment by Fikou** [comment]
cringe

**Comment by Deafult-Profile** [comment]
Ork

**Comment by Tharcoonvagh** [comment]
Cringe and bluepilled, lagging into a lava pool will kill you anyways

**Comment by IndieanaJones** [comment]
What the fuck is this
What happened that made you determine this was even close to necessary

**Comment by Deafult-Profile** [comment]
You can survive accidentally stepping on lava by using a medipen and spamming resist, playing miner while lagging makes fighting most of the megafauna near impossible.

**Comment by Deafult-Profile** [comment]
Miners haven't been fireproof for who knows how long, and they shouldn't start now just because someone died to an ash drake one too many times.

**Comment by TheMidnightRose** [comment]
Fire resistance doesnt matter to ash drakes, their fire attacks ignore armor. It will burn you through even Champion armor. Most of this is reliant on the miners having survival medipens to survive anything. Just ensure the cold resist is fixed, as right now every race BUT lizards dont take any damage from lavaland Watchers. Lowering the fire resist by one level or ensure that the explorer suit is burnable though with higher integrity would be a slightly better option.

**Comment by ExcessiveUseOfCobblestone** [comment]
*sees miner nerf*

miner players: 
image

**Comment by HomocidlePancakes** [comment]
Ashdrake breath deals damage on hit, along with setting you on fire, so *shrug.

**Comment by Deafult-Profile** [comment]
Back in my day, we had to use fire extinguishers when fighting drakes. Damn kids.

**Comment by Deafult-Profile** [comment]
Also only the tip of the ash drake's breath has armor pen, the rest is just fire. I've had no trouble at all beating the drake when the explorer suit didn't have fire resistance, are you guys alright???

**Comment by ATH1909** [comment]
Wait, if the original intent of this PR was to make it impossible to temporarily spacewalk with an explorer suit and a survival medipen, how does removing only the heat resistance (and not the cold resistance, 'cause Icebox is a thing) address that "problem" at all?

In other words, what's the point of this PR? Making lagging into lava even more punishing?

**Comment by Deafult-Profile** [comment]
ATH, obviously you're having some internet troubles, I understand. Not taking any damage from being set on fire makes lava barely punishing and makes the ash drake sink down to even easier than the hierophant. iirc lava is supposed to be an actual hazard and not something you can just pop a medipen and walk through.

**Comment by Deafult-Profile** [comment]
Also, this PR had always removed the explorer suit's heat resistance, did you even read it?

**Comment by wesoda25** [comment]
cringe comments

For feedback though, I don't get why you're removing all of its heat protection, just halve it or something. That way it isn't terrible in the way of fire/heat protection, but not as good as an actual fire suit or miner hardsuit.

**Comment by wesoda25** [comment]
> ATH, obviously you're having some internet troubles, I understand. Not taking any damage from being set on fire makes lava barely punishing and makes the ash drake sink down to even easier than the hierophant. iirc lava is supposed to be an actual hazard and not something you can just pop a medipen and walk through.

disingenuous

**Comment by Shadowflame909** [comment]
Bro...

You know all of the ash drakes attacks involve lava/fire...

Kind of stinky and makes fighting it impossible UNLESS you powergame it with some actual heat resistant armor

**Comment by Deafult-Profile** [comment]
I mean, everyone has to start somewhere, right? I honestly thought this was a bug at first.

**Comment by Deafult-Profile** [comment]
Looking back on it though, wesoda is definitely right, you take way too much damage if you let yourself heat up for a couple seconds. I'll try and change it so that you're given ample time to react and put yourself out

**Comment by ExcessiveUseOfCobblestone** [comment]
just don't get hit

**Comment by IndieanaJones** [comment]
Alright, since this isn't a shitpost evidently, a few things:
- This is going to make it harder on newer miners.  Have you actually playtested this change?  Do they feel right?  Also take into account server lag, lava accidents and being clipped by the drake should not be immediately fatal.
- Similarly, is this change worth making?  Does it improve the experience for somebody, I guess?
- You'll need to make sure you didn't imbalance the ash drake's lava attack.  As I'm sure you know, the ash drake's lava attack deals additional fire damage on top of exposing the target to lava, so it does more damage by a good margin than normal exposure.  What you'll need to check is that with your changes, this move doesn't incinerate them.

I still don't think this is a good idea, but you really haven't given any solid numbers as to why I should believe this is a valid issue, perhaps that could be something to add.

**Comment by wesoda25** [comment]
> This is going to make it harder on newer miners. Have you actually playtested this change? Does it feel right? Also take into account server lag, lava accidents and being clipped by the drake should not be immediately fatal.

Mining is supposed to be hard, its not a job for new players. 

> Similarly, is this change worth making? Does it improve the experience for somebody, I guess?

Yes, the people killed by ash drake loot bc the person just stayed on fire the whole fight

> You'll need to make sure you didn't imbalance the ash drake's lava attack. As I'm sure you know, the ash drake's lava attack deals additional fire damage on top of exposing the target to lava, so it does more damage by a good margin than normal exposure. What you'll need to check is that with your changes, this move doesn't incinerate them.

Don’t get hit

**Comment by Deafult-Profile** [comment]
also, shadowflame has revealed to me that he takes 3 survival medipens and 3 legion cores to beat current ash drake. Good luck if this change goes through.

**Comment by Deafult-Profile** [comment]
Okay so after some more testing with plasma and trit fires I've come to the conclusion that the problem is more the ash drake's fire. After I get some sleep I'll be changing that so it isn't as hot, this should make it so you only take only a small amount of burn damage when set on fire.

**Comment by IndieanaJones** [comment]
> Mining is supposed to be hard, its not a job for new players.

Sounds like gatekeeping to me.

**Comment by Deafult-Profile** [comment]
Alright, you don't take too much burn damage if you quickly extinguish yourself, but not putting the fire out is gonna hurt real bad.

**Comment by TheMidnightRose** [comment]
Honestly. Just drop the Explorer fire resist down from V to Fire IV. that should solve most of the problems and prevents massive rebalancing issues with Ash Drakes.

 If you are going to increase the lethality of lava/ash drake fire. Just ensure that peoples heads are not destroyed in the fire from them.

I honestly dont see much of a reason beyond this as I still see teams of miners die because one accidentally grabbed aggro on a megafauna because they didnt see it while mining. Last night in a round I got to see about eight people die to megafauna alone because the two miners died, then the other six went down to attempt to recover the miners.

**Comment by SweptWasTaken** [comment]
This player is a miner main so I feel like this has player bias.

**Comment by Blurboder** [comment]
good pr,totally not a biased review ^w^

**Comment by Akrilla** [comment]
femboy so +1

**Comment by Tumesuo** [comment]
femboy so +1

**Comment by Blurboder** [comment]
> femboy so +1

**Comment by imsxz** [comment]
i have 300h in mining, it was my most played job for about 2 years. this pr is fine and good. 
@TheMidnightRose that fire protection only applies to the suit getting destroyed by fire damage, which is negligible if you carry cloth on you to repair clothing(unless that was changed). it has nothing to do with the player getting damaged by fire. fire doesnt damage people. heat damages people, fire heats people up.

as it stands, you dont need fire extinguishers when fighting drake if you have survival pens. straight up the fire might as well not exist beyond the initial burn damage from the drakes breath(it does a flat 20 burn if you're hit by the projectile, unrelated to fire code)

**Comment by wesoda25** [comment]
> > Mining is supposed to be hard, its not a job for new players.
> 
> Sounds like gatekeeping to me.

Duh? New players should not be able to fight megafauna with a reasonable chance of success. They are meant to be hard, lavaland is not a hugbox.

**Comment by SweptWasTaken** [comment]
Unironically looks fine to me. @Deafult-Profile should probably request review from a maintainer for merge.

**Comment by ArcaneDefence** [comment]
image
Beat ash drake as a pacifist with 1 goliath plate and a watcher wing trophy.
Used I think 4 medipens, had to stop for 2 ash storms. 
Didn't even lose my wings.
Pretty sure there's plenty of room to make this harder, as I didn't have any issue with the fire tbh. Didn't notice my body temp going up at all, despite putting a minimum of effort in to extinguishing myself just to be confident I wouldn't burn my wings off.

Was also my first time fighting drake, so that's cool.

**Comment by Deafult-Profile** [comment]
The number of plates you have doesn't really matter since the ash drake is going to end up penetrating your armor anyway, nice job on your first drake kill though hug.

File: code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm [hunk 1]
```diff
@@ -271,7 +271,7 @@ Difficulty: Medium
 		if(istype(T, /turf/closed))
 			break
 		new /obj/effect/hotspot(T)
-		T.hotspot_expose(700,50,1)
+		T.hotspot_expose(500,50,1)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:274]: 1. Are these magic numbers (😢)

2. What is the purpose of this change

Comment (Deafult-Profile) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:274]: Makes drake fire not as hot so you don't take 60 burn damage even if you instantly put it out. I mean, it worked on my localhost.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:274]: what about Q1?

Comment (Deafult-Profile) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:274]: Yah, pretty sure they are.

**Comment by Deafult-Profile** [comment]
Sad!

**Comment by Rohesie** [comment]
Merge conflict.

**Comment by Deafult-Profile** [comment]
Fug it really has been open for that long

File: code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:274]: cringe, really should be a define

File: code/modules/mining/equipment/explorer_gear.dm [hunk 2]
```diff
@@ -20,9 +18,9 @@
 	icon_state = "explorer"
 	body_parts_covered = HEAD
 	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+	cold_protection = HEAD
 	min_cold_protection_temperature = FIRE_HELM_MIN_TEMP_PROTECT
-	max_heat_protection_temperature = FIRE_HELM_MAX_TEMP_PROTECT
-	armor = list("melee" = 30, "bullet" = 10, "laser" = 10, "energy" = 20, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 50, "wound" = 10)
+ 	armor = list("melee" = 30, "bullet" = 10, "laser" = 10, "energy" = 20, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 50, "wound" = 10)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mining/equipment/explorer_gear.dm:23]:
```suggestion
	armor = list("melee" = 30, "bullet" = 10, "laser" = 10, "energy" = 20, "bomb" = 50, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 50, "wound" = 10)
```

**Comment by imsxz** [comment]
cobby cumbrain

File: code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm [hunk 3]
```diff
@@ -4,6 +4,9 @@
 #define SWOOP_DAMAGEABLE 1
 #define SWOOP_INVULNERABLE 2
 
+#define DRAKE_FIRE_TEMP 500
+#define DRAKE_FIRE_EXPOSURE 50
```
Comment (bobbahbrown) [inline code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:8]: please document your code according to our [contribution standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine)

Comment (SweptWasTaken) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:8]: document my nuts

Comment (Deafult-Profile) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:8]: no

Comment (bobbahbrown) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:8]: image

**Comment by SweptWasTaken** [comment]
mum49VTHYg9N1co1x7WHdiw

File: code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm:8]:
```suggestion
/// Used whenever the drake generates a hotspot
#define DRAKE_FIRE_TEMP 500
/// Used whenever the drake generates a hotspot
#define DRAKE_FIRE_EXPOSURE 50
```

Comment (Deafult-Profile) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm]: only because bobbahbrown said I didn't know how to add comments

**Comment by SweptWasTaken** [comment]
bad first pr

**Comment by Deafult-Profile** [comment]
I have reported your comment, please do not make such disruptive posts on the tgstation repository

File: code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm (hunk 1 above)
Comment (Deafult-Profile) [reply code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm]: okie

**Comment by Shadowflame909** [comment]
2f8487abdb81e77f1f8f21c415e067a7
